dags/process_elife_zip_dag.py

```diff
@@ -212,6 +220,15 @@ def strip_related_article_tags_from_article_xml(**context) -> bytes:
     return etree.tostring(article_xml, xml_declaration=True, encoding='UTF-8')
 
 
+def add_missing_protocols(**context) -> bytes:
```

Comment:
would keep the URI terminology and call these `schemes` rather than `protocols`: https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#Examples

Reply:
updated